db/migrate/20240303144633_add_contact_info_to_donation_sites.rb

```diff
@@ -0,0 +1,9 @@
+class AddContactInfoToDonationSites < ActiveRecord::Migration[7.0]
+  def change
+    change_table :donation_sites do |t|
+      t.string :contact_name
+      t.string :email
+      t.string :phone
+    end
```

Comment:
```
  def change
    add_column :donation_sites, :contact_name, :string
    add_column :donation_sites, :email, :string
    add_column :donation_sites, :phone, :string
  end
```

Reply:
We went through this together on our call today and found:
* For Postgres at least, these are run in a transaction -- no worries about 
* Maaaybbbeee it locks the schema, but that shouldn't block reads/writes to the actual table
* Even if it does, we're not worries because our DB is small and we have a maintenance window when we do deploys
* But most importantly -- when run locally strong_migrations complains that it doesn't have visibility into this style of migration and that we have to validate it ourselves
* I'm going to open up a separate PR to get strong migrations to do this check as part of the CI tests, I'm surprised you didn't get an error!